src/main/java/com/vmware/vipclient/i18n/VIPCfg.java

```diff
@@ -120,9 +141,19 @@ public void initialize(String vipServer, String productName, String version) {
     }
     
     /**
-     * initialize the instance by a properties file
+     * Initialize VIPCfg instance using a configuration file
+     * 
+     * @param cfg The configuration file
+     * @param srcOpt The optional SourceOpt object which gives access to source messages
+     */
+    public void initialize(String cfg, SourceOpt srcOpt) throws VIPClientInitException {
```

Comment:
Where would you like to define the source? You hardcoded the source type to a ResourceBundle in getString2, which does not address the use case you mentioned the other time where source messages could come from DB or something else. 

What is your solution to this? Using the old getString with source parameter is not acceptable because it is error prone and devs have to type the source manually every time they need to use the string.